### ISSUE TITLE
Schedule and Zigbee cards do not load

### ISSUE THREAD

**Issue by msp1974** [asantaga/wiserHomeAssistantPlatform#521]
A change in v3.4.10 in the way the /wiser path is registered to support the custom cards shipped with this integration, does not correctly register the path in some cases causing the custom cards not to load.

Line 29 of 
```
custom_components/wiser/frontend/__init__.py needs amending to be:
```
```
await self.hass.http.async_register_static_paths(
    [StaticPathConfig(URL_BASE, pathlib.Path(__file__).parent, False)]
)
```

and pathlib added to the list of imports.

This will be fixed in v3.4.12

**Comment by msp1974** [comment]
Resolved in v3.4.12

**Comment by kernowkoi** [comment]
Had not noticed any issue with the schedule or Zigbee cards i 'think' was on 3.4.13,  
I have just updated to 3.4.14 and they don't load. has the bug come back?
